widget/ui/src/components/ExpandedRoute/ExpandedRoute.types.ts

```diff
@@ -6,7 +6,7 @@ import type { SwapInputProps } from 'src/containers/SwapInput/SwapInput.types';
 
 export interface PropTypes {
   steps: Step[];
-  tags: RouteTag[];
+  sortedQuoteTags: RouteTag[];
```

Comment:
I'm not ok with this change. It's just a component and the user can pass sorted or unsorted tags.

Reply:
Fixed